xml/installation-installation-ironic-install_ironic_overview.xml

```diff
@@ -2,17 +2,16 @@
 <!DOCTYPE chapter [
  <!ENTITY % entities SYSTEM "entities.ent"> %entities;
 ]>
-
 <chapter xml:id="install_ironic_overview"
  xmlns="http://docbook.org/ns/docbook" version="5.1"
  xmlns:xi="http://www.w3.org/2001/XInclude"
  xmlns:xlink="http://www.w3.org/1999/xlink">
  <title>Installing Baremetal (Ironic)</title>
  <para>
-  Bare Metal as a Service is enabled in this release for deployment of nova
-  instances on bare metal nodes using flat networking. HPE DL and SL line of
-  servers are supported.
-  <!-- FIXME: HPE mention etc. -->
+  Bare Metal as a Service is enabled in this release for deployment of Nova
+  instances on bare metal nodes using flat networking.
+  <phrase vendor="hpe">&vendor;</phrase> DL and SL line of servers are
```

Comment:
This is a rather novel approach to profiling, as the output is the following:

* HPE: `[...] networking. HPE DL and SL line [...]`
* SUSE: `[...] networking. DL and SL line [...] `

I other words, yes, you removed the HPE mention in the SUSE documentation but it does not make any sense anymore. Depending on the circumstances (i.e. what the product team is saying), you could do one of the following:

* remove the whole sentence in the SUSE version (because in general we don't necessarily push our customers to buy HPE hardware over say Dell or Lenovo).
* just leave the sentence as is, but maybe use the `&hpe;` entity (without any profiling!!)

Note that you have done added this weird version of `&vendor;` in a lot of places in this PR and I have not added a comment for each occurrence.